pages/common/tldr.md

```diff
@@ -9,12 +9,16 @@
 
 - Show the tar tldr page for Linux:
 
-`tldr -p {{linux}} {{tar}}`
+`tldr --platform {{linux}} {{tar}}`
 
 - Get help for a Git subcommand:
 
 `tldr {{git-checkout}}`
 
 - Update local pages (if the client supports caching):
```

Comment:
We can explain the short argument:
```suggestion
- [u]pdate local pages (if the client supports caching):
```

Reply:
I prefer the original version in this case. For me, it breaks with the structure of the rest of the page

Reply:
I also prefer the original version. Also, the `-u` is pretty intuitive and self explanatory IMO.

Reply:
I'm not sure this change adds any value. The existing version looks fine.

Note also this from the client spec:

> Clients MAY choose to only implement the short version of an option, ignoring the long form.